setup.py

```diff
@@ -5,6 +5,11 @@
 with open("requirements.txt") as fh:
     install_requires = fh.readlines()
 
+with open("test-requirements.txt") as fh:
+    test_install_requires = fh.readlines()
+    test_install_requires.remove("-r requirements.txt")
```

Comment:
Let's document why this is needed explicitly, to not confuse folks.

Reply:
It is putting the packages in the "test-requirements.txt" into a list, but then it removes "-r requirements.txt". However, the packages in requirements.txt are added in the next line with `test_install_requires.extend(install_requires)`. All this is because pypi doesn't know how to read a given file to retrieve packages.

added directly to the file too